components/definitions/rewardsInternals.d.ts

```diff
@@ -82,4 +89,10 @@ declare namespace RewardsInternals {
     entries: AdDiagnosticsEntry[]
     diagnosticId: string
   }
+
+  export enum Environment {
```

Comment:
Instead of having the mappings in TS, I think there's a way to use mojo. If you move the `Environment` enum from `ledger.mojom` to `ledger_types.mojom`, you should be able to bring in the enum names. See `internals/components/externalWallet.tsx` for an example.

To bring in the mojom enum in this TS declaration file, you have to use the ugly syntax above (line 58).